src/buildHTML.js

```diff
@@ -1473,6 +1501,230 @@ groupTypes.accent = function(group, options) {
     }
 };
 
+groupTypes.horizBrace = function(group, options) {
+    const style = options.style;
+    const ruleWidth = fontMetrics.metrics.defaultRuleThickness /
+        style.sizeMultiplier;
+
+    const haveNote = (group.type === "supsub");
+    let note;
+    let newOptions;
+    let noteReset;
+    if (haveNote) {
+        if (group.value.sup) {
+            newOptions = options.withStyle(style.sup());
+            note = buildGroup(group.value.sup, newOptions);
+            noteReset =  makeSpan([style.reset(), style.sup().cls()],
+                [note], newOptions);
+        } else {
+            newOptions = options.withStyle(style.sub());
+            note = buildGroup(group.value.sub, newOptions);
+            noteReset =  makeSpan([style.reset(), style.sub().cls()],
+                [note], newOptions);
+        }
+        group = group.value.base;
```

Comment:
Say that a author writes `\overbrace{long equation}^{short note}`. Then `group.type` will be "supsub" and `group.value.base` corresponds to `long equation`.